core/cmd/client_test.go

```diff
@@ -195,17 +202,21 @@ func TestFileAPIInitializer_InitializeWithoutAPIUser(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			db := pgtest.NewSqlxDB(t)
 			orm := sessions.NewORM(db, time.Minute, logger.TestLogger(t))
-			// Clear out fixture user
-			orm.DeleteUser()
+
+			// Clear out fixture users/users created from the other test cases
+			// This asserts that on initial run with an empty users table that the credentials file will instantiate and
+			// create/run with a new admin user
+			_, err := db.Exec("DELETE FROM users;")
+			require.NoError(t, err)
```

Comment:
This is a bad hack really, we shouldn't be clobbering fixtures from a transactional test because it can cause very long locking delays. Ideally we'd spin up a new database and run on there, or adjust all the other tests to insert a random user instead of using the fixture one.

Not something I expect you to fix in this PR since its just copying the existing test.

You can do this though:

```suggestion
			// Clear out fixture users/users created from the other test cases
			// This asserts that on initial run with an empty users table that the credentials file will instantiate and
			// create/run with a new admin user
			pgtest.MustExec(t, db, "DELETE FROM users;")
```

Reply:
Yes definitely, agreed. 

That's a nice simplification though, updated all 🙂